src/components/Global/AccountItem.tsx

```diff
@@ -0,0 +1,116 @@
+import {Check} from "lucide-react-native";
+import React from "react";
+import {useTheme} from "@react-navigation/native";
+
+const AccountItem: React.FC<{ account: PrimaryAccount, additionalStyles?: StyleProp<ViewStyle>, endCheckMark: boolean }> = ({
+  account,
+  additionalStyles,
+  endCheckMark
+}) => {
+  const theme = useTheme();
+  return (
+    <View
+      style={additionalStyles ? {
+        // @ts-expect-error
+        ...additionalStyles,
+        flexDirection: "row",
+        padding: 9,
+        borderStyle: "solid",
+        alignItems: "center",
+      }: {
+        flexDirection: "row",
+        padding: 9,
+        borderStyle: "solid",
+        alignItems: "center",
+      }}
+    >
+      <View
+        style={{
+          width: 30,
+          height: 30,
+          borderRadius: 80,
+          backgroundColor: "#000000",
+          marginRight: 10,
+        }}
+      >
+        <Image
+          source={account.personalization.profilePictureB64 ? { uri: account.personalization.profilePictureB64 } : defaultProfilePicture(account.service, account.identityProvider?.name || "")}
+          style={{
+            width: "100%",
+            height: "100%",
+            borderRadius: 80,
+          }}
+          resizeMode="cover"
+        />
+      </View>
+      <View
+        style={{
+          flexDirection: "column",
+          gap: 2,
+        }}
+      >
+        <View style={{ flexDirection: "row", flexWrap: "nowrap", minWidth: "90%", maxWidth: "75%" }}>
+          <Text
+            style={{
+              fontSize: 16,
+              fontFamily: "semibold",
+              color: theme.colors.text,
+              flexShrink: 1
+            }}
+            numberOfLines={1}
+            ellipsizeMode="tail"
+          >
+            {account.studentName?.first || "Utilisateur"}{" "}
+            {account.studentName?.last || ""}
+          </Text>
+        </View>
+
+        <Text
+          style={{
+            fontSize: 15,
+            fontWeight: 500,
+            color: theme.colors.text + "50",
+            fontFamily: "medium",
+            maxWidth: "70%",
+          }}
+          numberOfLines={1}
+          ellipsizeMode="tail"
+        >
+          {AccountService[account.service] !== "Local" ?
+            AccountService[account.service] :
+            account.identityProvider ?
+              account.identityProvider?.name :
+              "Compte local"
+          }
+        </Text>
+      </View>
+      {endCheckMark &&
+              <Reanimated.View
+                style={{
+                  position: "absolute",
+                  right: 15,
+                }}
+                entering={animPapillon(ZoomIn)}
+                exiting={FadeOut.duration(200)}
+              >
+                <Check
+                  size={22}
+                  strokeWidth={3.0}
+                  color={theme.colors.primary}
+                />
+              </Reanimated.View>
```

Comment:
Tu as un décalage d'indentation

Reply:
ajoute une `(` après le `&&`

```suggestion
        <Reanimated.View
          style={{
            position: "absolute",
            right: 15,
          }}
          entering={animPapillon(ZoomIn)}
          exiting={FadeOut.duration(200)}
        >
          <Check
            size={22}
            strokeWidth={3.0}
            color={theme.colors.primary}
          />
        </Reanimated.View>
```
et `)` avant le `}`